src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -95,7 +95,13 @@ class InstrumentCodeAction implements Action<Task> {
                 }
 
                 getMethodContexts().each {
-                    ant.methodContext(name: it.name, regexp: it.regexp)
+                    ant.methodContext(
```

Comment:
How do you want to re-model the CloverContextConvention?  Should there be two different classes with no relation, one for statements, and one for methods?  Or should there be a class hierarchy?  I'm leaning towards the first.